metadata/service/server/cluster.yml

```diff
@@ -0,0 +1,10 @@
+applications:
```

Comment:
There's no watchdog cluster, so don't have this file.

Reply:
Fixed...